cmd/sample.go

```diff
@@ -14,17 +14,21 @@ number of lines, allowing us to manually test the process quicker.`
 var (
 	maxLines  int
 	targetDir string
+	updatedAt string
 )
 
 var sampleCmd = &cobra.Command{
 	Use:   "sample",
 	Short: "Creates sample data of the source files from the Federal Revenue",
 	Long:  sampleHelper,
-	RunE: func(_ *cobra.Command, _ []string) error {
+	RunE: func(cmd *cobra.Command, _ []string) error {
 		if err := assertDirExists(); err != nil {
 			return err
 		}
-		return sample.Sample(dir, targetDir, maxLines)
+
+		mockUpdatedAt := cmd.Flags().Changed("mock-updated-at")
+
+		return sample.Sample(dir, targetDir, maxLines, updatedAt, mockUpdatedAt)
```

Comment:
Qual a vantagem de chamar o `Changed`?


```suggestion
		return sample.Sample(dir, targetDir, maxLines, updatedAt, updatedAt)
```

Reply:
O `Changed` retorna se a flag foi setada pelo usuário. Mas acho que entendi o seu ponto:
- Se foi setada, o valor `updated_at` vai ter mudado de vazio para alguma coisa, então no caso seria só fazer um `if len(updatedAt) > 0 { ... }`

É isso?

Reply:
Não — ou quase:
  * depende de onde vai esse `if` (na minha opinião, ele vai no `sample/sample.go`)
   *e não precisa ser `len(…)`, como o valor padrão (_zero value_) de `string` em Go é `""` o melhor é fazer `if updatedAt != ""`